src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -182,7 +182,7 @@ const UsersFragment = ({
   const renderUsersItem = () => {
     if (isFirstTimeOpen) {
       if (withoutRecent) {
-        if (initialUsers.length !== 0) {
+        if (initialUsers.length !== 0 || route.name === 'TopicMemberScreen') {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new condition added to the `if` statement checks if the route name is `'TopicMemberScreen'`. This change could potentially cause the function `renderUsersItem` to return even when `initialUsers` array is empty. If the function `renderDiscoveryItem` does not handle an empty `item`, this could lead to unexpected behavior or errors. Please ensure that `renderDiscoveryItem` can handle an empty `item` or consider adding additional checks before calling `renderDiscoveryItem`.
```diff
- if (initialUsers.length !== 0 || route.name === 'TopicMemberScreen') {
+ if ((initialUsers.length !== 0 && item) || route.name === 'TopicMemberScreen') {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->